subscribe_juniper_junos.go

```diff
@@ -232,7 +232,7 @@ func subscribeJunos(conn *grpc.ClientConn, jctx *JCtx, cfg Config, paths []Paths
 	//cfg := &jctx.config
 	for i := range paths {
 		var pathM na_pb.Path
-		pathM.Path = cfg.Paths[i].Path
+		pathM.Path = paths[i].Path
 		pathM.SampleFrequency = uint32(cfg.Paths[i].Freq)
```

Comment:
Should we pick sampleFrequenct also from paths[i] ? How about other places where we pick from cfg.Paths[i] ?

Reply:
Oh I see it is already merged. If this is an issue, maybe you can raise an other pull request.

Reply:
only at this location it is missed. Rest are taken care of
Will raise a new pull request